projects/codebar-pairing.md

```diff
@@ -0,0 +1,40 @@
+# Improve codebar's Pairing Process through Automation
+
+## About
+
+* Who is it for?
+
+codebar: https://codebar.io/
+
+codebar is a nonprofit organisation whose primary aim is to facilitate the growth of a more diverse tech community. We run regular free programming workshops and all day events for individuals underrepresented in the tech industry, to make coding more accessible to them and to expand their career opportunities. There are currently 20 codebar chapters around the world with 11,000 members, including students and mentors.
+
+We've run over 900 workshops since starting in 2013. At our workshops we pair students and mentors one to one or two to one depending on the numbers of students and coaches attending each workshop and the level of difficulty of the projects being worked on.
+
+* What problem is it trying to solve?
+
+Our current process of organising our signed in student list (in preparation for pairing students with coaches) is almost entirely manual and quite time consuming, even small gains in automation would be beneficial.
+
+To that end, see:
+https://github.com/codebar/planner/pull/823
+
+* What was the motivation behind?
+
+Over the last year, the number of attendees at our weekly Wednesday central London workshops has been steadily increasing. This is an intentional shift, as we have opened up more places for each workshop, and it's great to be able to accommodate more individuals at each workshop, but it has made our current pairing process feel quite hectic.
+
+## Links
+
+The codebar planner app that we use for managing workshops and signing in students and coaches can be found here: https://github.com/codebar/planner
+
+## Acknowledgement
+
+It is likely that Google Sheets will be quite helpful in making the codebar pairing process smoother and easier.
+
+Here is an interesting talk on Google Sheets delivered recently at Google Cloud Next London: https://cloud.withgoogle.com/next18/london/sessions/session/243742
+
+## Contributors
+
+Last, but not least, names of the proud contributors please!
+
+| Name             | GitHub  | Twitter    | Medium         | Other |
+| :--------------- | :------ | :--------- | :------------- | :---- |
```

Comment:
Looks good, just 1 suggestion: 

The width of the columns will keep changing depending on the content, this will become difficult to maintain and make future contributions less clear. I would suggest keep it to the minimum amount which is 3 dashes.

> Cells can vary in width and do not need to be perfectly aligned within columns. There must be at least three hyphens in each column of the header row.

from https://help.github.com/articles/organizing-information-with-tables/